App/localization/en/index.ts

```diff
@@ -0,0 +1,85 @@
+const translation = {
+  "Global": {
```

Comment:
Everything looks good to me, the only comment I have is to use a consistent case convention. https://lokalise.com/blog/translation-keys-naming-and-organizing/#1_Develop_a_proper_naming_scheme